.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  setup:
+    docker:
+      - image: node:10.13.0
+    steps:
+      - checkout
+      - run:
+          name: Initial Setup
+          command: |
+            curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+            python get-pip.py
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+            export PATH="$HOME/.yarn/bin:$PATH"
+            python --version
+            python3 --version                     
+            pip --version          
+
+      - run:
+          name: Installing Dependencies
+          command: |
+            yarn install --frozen-lockfile
+            pip install --user -r requirements.txt
+
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+            - '.cache'
+            - '.eslintcache'
+
+      - run:
+          name: Running Scripts
+          command: |
+            yarn prettier --list-different
+            yarn lint
+            git config --global user.email "nihal9ns@gmail.com" # add your github email here
+            git config --global user.name "nihal9ns"    # add your github user name here
+
+      - run:
+          name: Running Tests
+          command: |
+            yarn test-schema
+            yarn build              
+            yarn jest --updateSnapshot --runInBand
+            yarn test-e2e  
+
+      - run:
+          name: Getting Code Coverage
+          command: |
+            yarn codecov
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - setup:
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
So it will run on master plus any PR?

Reply:
For PR we need to configure our project settings from the web interface. Refer the doc I added.